### DISCUSSION TITLE
After cutting the picture, the annotation information is disordered and incorrect

### DISCUSSION THREAD

**Discussion by zxx20231119** [obss/sahi#1065] [Q&A]
![bwk_6_3_0_524_2500_3024](https://github.com/user-attachments/assets/d225da00-4c33-4729-bb5e-5a3ba6da9121)

thanks！

**Comment by zxx20231119** [comment]
![image](https://github.com/user-attachments/assets/3e8d6c07-3c08-4d97-8aa2-6ee5ce642fd3)

this is source annotation pic

**Comment by fcakyon** [comment]
Please provide the script/function you are using, a sample annotation, and an image file, @zxx20231119. This information is necessary for us to identify the source of the issue.

**Comment by zxx20231119** [comment]
Thank you for your reply.
sahi:0.11.18
code:
from sahi.scripts.slice_coco import slice
    
slice(
    image_dir="./bwk/images",
    dataset_json_path="./bwk/coco.json",
    output_dir="./bwk_out",
    slice_size=1500,
    overlap_ratio=0.25, )

[ceshi_data.zip](https://github.com/user-attachments/files/16191867/ceshi_data.zip)

**Comment by zxx20231119** [comment]
Hello, is there a problem with this data?
